web/src/main/java/org/springframework/security/web/header/writers/DelegatingRequestMatcherHeaderWriter.java

```diff
@@ -27,12 +28,13 @@
  * {@link RequestMatcher#matches(HttpServletRequest)} returns true.
  *
  * @author Rob Winch
+ * @author Ankur Pathak
  * @since 3.2
  */
 public final class DelegatingRequestMatcherHeaderWriter implements HeaderWriter {
 	private final RequestMatcher requestMatcher;
 
-	private final HeaderWriter delegateHeaderWriter;
+	private final CompositeHeaderWriter compositeHeaderWriter;
```

Comment:
Let's not change this class. It'll support composites just fine without an update.